typescript/package.json

```diff
@@ -22,6 +22,8 @@
   },
   "homepage": "https://github.com/commontoolsinc/labs#readme",
   "devDependencies": {
+    "@bytecodealliance/componentize-js": "^0.10.1",
+    "@bytecodealliance/jco": "^1.3.1",
```

Comment:
These two bytecodealliance packages are used by the build server (via shell exec), hence the global install earlier. TIL we are also using them in e.g. `typescript/common/data/package.json` as well to generate types. I'm not sure there's a way to synchronize these dependencies between those worlds, at the very least we should try and target the same versions (in lieu of a better synchronization mechanism, maybe just comments that they should be synchronized).

Note that componentize-js v0.10 had issues in the builder that we had to rollback; for consistency purposes, consider using the same version (< 0.10) here https://github.com/commontoolsinc/system/issues/45

Reply:
FWIW this synchronization problem will be "solved" if #63 pans out

Reply:
(of course, we'll have traded it for a new problem: monolithic dependency management via Nix :sweat_smile:  )

Reply:
... perhaps a poison chalice or a panacea.

but that's good information for me to experiment with @jsantell